packages/react/src/components/configure-adapter/configure-adapter.tsx

```diff
@@ -58,6 +57,7 @@ const useAppliedAdapterArgsState = ({
   const [appliedAdapterArgs, setAppliedAdapterArgs] =
     useState<TAdapterArgs>(initialAdapterArgs);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
```

Comment:
I didn't want to change the deps here because that could change the behavior and I didn't want to risk it now